x-pack/plugins/observability/server/lib/rules/custom_threshold/register_custom_threshold_rule_type.ts

```diff
@@ -79,27 +78,9 @@ export function thresholdRuleType(
     timeSize: schema.number(),
   };
 
-  const nonCountCriterion = schema.object({
-    ...baseCriterion,
-    metric: schema.string(),
-    aggType: oneOfLiterals(METRIC_EXPLORER_AGGREGATIONS),
-    metrics: schema.never(),
-    equation: schema.never(),
-    label: schema.never(),
-  });
-
-  const countCriterion = schema.object({
-    ...baseCriterion,
-    aggType: schema.literal('count'),
-    metric: schema.never(),
-    metrics: schema.never(),
-    equation: schema.never(),
-    label: schema.never(),
-  });
-
   const customCriterion = schema.object({
     ...baseCriterion,
-    aggType: schema.literal('custom'),
+    aggType: schema.maybe(schema.literal('custom')),
```

Comment:
Added as `maybe`, otherwise rules that were created before will throw a validation error.

```
observability.rules.custom_threshold:7375e097-7250-4d2d-b69b-5af710b96379: 
execution failed - params invalid: [criteria.0.aggType]: definition for this key is missing
```